client/components/organisms/FaultReporting/FaultReporting.js

```diff
@@ -0,0 +1,332 @@
+import React, { useState, useEffect } from "react";
```

Comment:
why is the registration page here?

Reply:
I deleted this folder - don't need this page